onedal/spmd/basic_statistics/basic_statistics.py

```diff
@@ -23,4 +25,13 @@
 class BasicStatistics(BaseEstimatorSPMD, BasicStatistics_Batch):
     @support_usm_ndarray()
     def compute(self, data, weights=None, queue=None):
+        warnings.warn(
+            "Method `compute` was deprecated in version 2024.3 and will be "
+            "removed in 2024.5. Use `fit` instead."
```

Comment:
please update in which exact version it will be removed

Reply:
Based on the answer https://github.com/intel/scikit-learn-intelex/pull/1644#discussion_r1652781106 make sense to remove warning from here.